docs/application/dotnet/guides/nui/animation-types.md

```diff
@@ -1,127 +1,45 @@
-_keyFrames.Add(0.3f, new Size(window.Size.Width * 0.7f, window.Size.Height * 0.7f, 0.0f));
-_keyFrames.Add(1.0f, new Size(window.Size));
-
-KeyFrames keyFrames = KeyFrames::New();
-keyFrames.Add( 0.0f, Vector3( 10.0f, 10.0f, 10.0f ) );
-keyFrames.Add( 0.7f, Vector3( 200.0f, 200.0f, 200.0f ) );
-keyFrames.Add( 1.0f, Vector3( 100.0f, 100.0f, 100.0f ) );
-```
-
-Next, you can add the key frames to your animation:
-
-```
-_animation.AnimateBetween(_imageView, "Size", _keyFrames, 4000, 6000, Animation.Interpolation.Linear);
-```
-
-When you play the animation, NUI animates the position of `view1` between the specified key frames. `view1` animates from (10.0f, 10.0f, 10.0f) to (200.0f, 200.0f, 200.0f) for 70% of the animation time, and spends the remaining time animating back to (100.0f, 100.0f, 100.0f).
-
-The advantage of specifying a key frame at 0% is that regardless of the position of `view1`, it starts from (10.0f, 10.0f, 10.0f). If `AnimateTo()` is used, then the start position is the current position of `view1`.
-
-The following comprehensive example of using key frame is taken from `FocusEffect.cs`:
-
-```
-focusData.ImageItem.Size = new Size(100.0f, 100.0f, 0.0f);
-parentItem.Add(focusData.ImageItem);
-
-Size targetSize = focusData.TargetSize;
-Size initSize = focusData.InitSize;
-
-KeyFrames keyFrames = new KeyFrames();
-
-keyFrames.Add(0.0f, initSize);
-keyFrames.Add(focusData.KeyFrameStart, initSize);
-keyFrames.Add(focusData.KeyFrameEnd, targetSize);
-
-// For halo add an extra key frame to shrink it (in 20% of time after it has finished)
-if (focusData.Name == "halo")
-{
-   keyFrames.Add(focusData.KeyFrameEnd + 0.2f, initSize);
-}
-
-_animation.AnimateBetween(focusData.ImageItem, "Size", keyFrames, Animation.Interpolation.Linear, new AlphaFunction(AlphaFunction.BuiltinFunctions.EaseOutSine));
-```
-
-<a name="2"></a>
 ## Path Animation
 
-The `Path` class can be used to animate the position and orientation of actors. (a three-dimentional parametric curve)
+The `Path` class is a three-dimensional parametric curve that is used to animate the position and orientation of actors.
 
-The black points are the points where the 'dali' logo travels. The red points are the control points which express the curvature of the path on the black points.
+The `Path` consists of several points and each adjacent pair of points becomes a segment of the path. Each segment can be curved by adding control points to it.
 
-**Figure: Path animation**
+In the following example, the P1, P2 and P3 form the path where the blue circle travels. The C1 and C2 are the control points for a first segment `[P1-P2]`, where C3 and C4 are for a second segment `[P2-P3]`. The control points express the curvature of the path.
 
-![Path animation](./media/path_animation.png)
+<div style="text-align:center;width:100%;"><img src="./media/path.svg" /></div>
 
-The following code presents the black points:
+To generate the `Path' object, use the following code:
```

Comment:
Just one more check before merging. 
@divyk , `Path' is not in code tag style and just quotation marks are used. Is it intended?

![image](https://user-images.githubusercontent.com/9958374/70595127-d9b35e00-1c25-11ea-8011-3e887540cb59.png)

Reply:
@mijin85cho , thanks I missed seeing it.

@rabbitfor , please make this small change:

To generate the `Path` object, use the following code: